bin/test-example

```diff
@@ -50,7 +58,9 @@ if [ "$exampletype" = "success" ]; then
     # Implicit exit value: this is last thing in this path,
     # so the script's exit value = stack's exit value.
     # If statements are reordered, please preserve this property.
-    runstack "test"
+    if ! runstack "test"; then
+        exit 1
+    fi
```

Comment:
@petertseng, could you please take a look at these changes in the test scripts when you get some time. I have some kind fever right now, so my brain isn't 100%, and I don't want to merge anything that could break the tests scripts you wrote.